frontend/src/pages/user/home/Dashboard/Dashboard.module.css

```diff
@@ -1,8 +1,8 @@
 .userAvatar {
-  background-color: #E6F7FF;
-  width: 100%;
-  height: auto;
-  aspect-ratio: 1 / 1;
+  background-color: #E6F7FF !important;
```

Comment:
- would it not work without `!important!`? (to all related)
- Although `!important` would fix some style issues. I would not advice to do this all the time most especially if its really not needed.